builtins/safe-default-configuration.txt

```diff
@@ -397,33 +389,24 @@ svg marker
 - clip-rule
 - color
 - color-interpolation
-- color-interpolation-filters
```

Comment:
Shouldn't we list these as purposely omitted as well? We do it for overflow below.

Reply:
Good idea.